data_generators/random_match.py

```diff
@@ -1,5 +1,6 @@
 from random import sample
 from typing import Dict, List
+import pandas
```

Comment:
Seems like this import isn't needed

Reply:
Thanks for the review, Michael.  DataFrame was used from the pandas library.